CHANGELOG.md

```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Add snaps search ([#14100](https://github.com/MetaMask/metamask-extension/pull/14100))
-- Dark mode: Updating error color and adding story ([#14116](https://github.com/MetaMask/metamask-extension/pull/14116))
-- Fix Settings Header Back Button and update "Add Contacts" position ([#14103](https://github.com/MetaMask/metamask-extension/pull/14103))
-- Dark mode: Adding wyre, transak and deposit eth logo ([#13953](https://github.com/MetaMask/metamask-extension/pull/13953))
-- Dark mode: Removing double and single arrows svg icons for confirm-page-container-navigation ([#14136](https://github.com/MetaMask/metamask-extension/pull/14136))
-- Removing unused SUCCESS, ERROR and ALERT colors from codebase ([#14122](https://github.com/MetaMask/metamask-extension/pull/14122))
-- Replacing close icon with font awesome version in settings-search.js ([#14131](https://github.com/MetaMask/metamask-extension/pull/14131))
-- Dark mode: icons ens input ([#14133](https://github.com/MetaMask/metamask-extension/pull/14133))
-- Dark Mode: Replace color occurences ([#14112](https://github.com/MetaMask/metamask-extension/pull/14112))
-- Dark Mode: Permissions container ([#14077](https://github.com/MetaMask/metamask-extension/pull/14077))
-- Dark Mode: Encrypt / Decrypt ([#14081](https://github.com/MetaMask/metamask-extension/pull/14081))
-- Dark Mode: Onboarding Flow ([#14125](https://github.com/MetaMask/metamask-extension/pull/14125))
-- Dark Mode: Make list item text consistent ([#14142](https://github.com/MetaMask/metamask-extension/pull/14142))
-- Dark Mode: Remove Account modal ([#14127](https://github.com/MetaMask/metamask-extension/pull/14127))
-- Dark mode: Replacing svg image with font awesome icon for main-quote-summary ([#14137](https://github.com/MetaMask/metamask-extension/pull/14137))
-- Dark mode: transaction activity log icons ([#14139](https://github.com/MetaMask/metamask-extension/pull/14139))
-- Dark mode: Removing connected svg from account menu in favour of fontawesome ([#14140](https://github.com/MetaMask/metamask-extension/pull/14140))
-- Dark Mode:  Settings -> Networks ([#14134](https://github.com/MetaMask/metamask-extension/pull/14134))
-- Dark Mode:  New Account modal ([#14132](https://github.com/MetaMask/metamask-extension/pull/14132))
-- Dark Mode: Cancel Transaction ([#14129](https://github.com/MetaMask/metamask-extension/pull/14129))
-- Dark Mode: Select Hardware ([#14086](https://github.com/MetaMask/metamask-extension/pull/14086))
-- Bump @metamask/design-tokens from 1.4.2 to 1.4.4 ([#14119](https://github.com/MetaMask/metamask-extension/pull/14119))
-- Sync `master` with `develop` ([#14120](https://github.com/MetaMask/metamask-extension/pull/14120))
-- Add token list to token details page ([#14051](https://github.com/MetaMask/metamask-extension/pull/14051))
-- Dark mode: colors obj UI, GREY, WHITE ([#14118](https://github.com/MetaMask/metamask-extension/pull/14118))
+    - "Send max" doesn't work on Optimism
```

Comment:
`"Send max" doesn't work on Optimism` -> `Fix 'Send max' button when on the Optimism network`